pkg/controllers/dynakube/activegate/internal/customproperties/conditions.go

```diff
@@ -0,0 +1,7 @@
+package customproperties
+
+const customPropertiesConditionType string = "CustomProperties"
+
+func customPropertiesConditionTypeString(customPropertiesOwnerName string) string {
+	return customPropertiesConditionType + "-" + customPropertiesOwnerName
```

Comment:
this is not super related to your change, but what is this `customPropertiesOwnerName` and why do we care 😅 ?
I know its based on the service-account to be used by the activegate, but yet again, why do we care ?

1 dynakube will create 1 activegate (from the activegate section) so that postfix means nothing, just looks weird (`Type: CustomProperties-kubernetes-monitoring`), also it wouldn't work at all if we would deploy multiple activegates from 1 dynakube

I wouldn't change the behaviour of the old code related to this (it's not related to your intended change), BUT I would definitely not have it in new code. (ie.: the conditions), just have the `Type` be `CustomPropertiesSecret`

Reply:
You are right, will remove it 👍